### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by Benji377** [Benji377/OpenMusic#136] [Announcements]
This has been a long-awaited release now. Even if the app is not at the state I would like it to be, this release was really necessary to set a checkpoint and refresh the latest stable release which is v1.1 and at this point in time completely outdated.
A lot of stuff has changed and this time also visually. But before I go into that, I would like to thank everyone who helped me out with this project. It's my first collaboration so far and I am proud of how it's turning out. Special thanks go to @squivix which contributed a lot to this project, I think he almost contributed more than myself. And special thanks also go to @SilviaJack28 for making those awesome backgrounds you can see in the app. Maybe she will do some more in the future, who knows ;D

What's new?
- Settings option almost completely implemented
- New themes and colors to chose from
- You can now choose the folder from where the songs are loaded from
- It is possible to swipe left and right to change songs
- By long-clicking on the dedicated buttons the song skips 10 seconds forwards or back

There were also other new additions and bug fixes, but you can look at it yourself by downloading the app for free.

<hr /><em>This discussion was created from the release <a href='https://github.com/Benji377/SocyMusic/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by squivix** [comment]
Congratulations, Benji! Sorry I've been inactive lately. I'm focused on learning web dev right now and also preparing for next college semester.

**Reply by Benji377** [reply]
Thank you! And don't worry about taking some breaks, we all need them. It's impressive how productive you are and good luck with your college semester.
